.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+## Description 
+
+Describe the changes or additions included in thiis PR.
+
+## Test Plan 
+
+How did you test the new or updated feature?
+
+---
+If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes seection as hightlighted during the release process
+
+### Type of Change (Check all that apply)
+
+- [ ] user-visible impact
+- [ ] breaking change for a client SDKs
+- [ ] breaking change for FNs (FN binary must upgrade)
+- [ ] breaking change for validators or node operators (must upgrade binaries)
+- [ ] breaking change for on-chain data layout
+- [ ] necessitate either a data wipe or data migration
+
+### Persona affected
+
+Reference: [go/sui-persona](http://go/sui-persona)
```

Comment:
Do we care if people outside of Mysten can see this link?

Reply:
If there are any internal only links we should strip them

Reply:
@srinath-mysten has requested this information be listed in the template. @srinath-mysten thoughts?

Reply:
Its sort of useless to have anything that is a private link in the public repo

Reply:
Ok, let's remove this for now.